### PR TITLE
Testing with rspec for project Tic-Tac-Toe

### PR DESCRIPTION
**Pull request by rubenpazch**
+ Add specs for all public methods you created in Tic Tac Toe project.
+ Use existing repositories for Tic Tac Toe and open new Pull Requests.
+ Create a feature branch for testing project 
+ Wrap specs for each method in describe block
+ Instead of using before_each to define a variable for the test cases, use let.
+ Use one it block for each expected behaviour. Do not test multiple aspects in one it block.
+ Use your common sense: test positive and negative scenarios.

### CODE REVIEW

File: spec/board_spec.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
+require 'rspec'
```
Comment (kobiyoyo) [inline spec/board_spec.rb]: You don't need this, lets remove this

File: spec/game_spec.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
+require 'rspec'
```
Comment (kobiyoyo) [inline spec/game_spec.rb]: You don't need this, lets remove this

File: spec/player_list_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+require 'rspec'
```
Comment (kobiyoyo) [inline spec/player_list_spec.rb]: You don't need this, lets remove this

File: spec/player_spec.rb [hunk 4]
```diff
@@ -0,0 +1,59 @@
+require 'rspec'
```
Comment (kobiyoyo) [inline spec/player_spec.rb]: You don't need this, lets remove this

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team     
#### Great Job so far, but there are a few more things required. 
- [x] Lets write test for when there is a draw between both players

### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

File: spec/game_spec.rb (hunk 2 above)
Comment (enelesmai) [reply spec/game_spec.rb]: Fixed, thanks for the suggestion!

File: spec/player_list_spec.rb (hunk 3 above)
Comment (enelesmai) [reply spec/player_list_spec.rb]: Fixed, thanks for the suggestion!

File: spec/player_spec.rb (hunk 4 above)
Comment (enelesmai) [reply spec/player_spec.rb]: Fixed, thanks for the suggestion!

File: spec/board_spec.rb (hunk 1 above)
Comment (enelesmai) [reply spec/board_spec.rb]: Fixed, thanks for the suggestion!

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team ,  You have done a great job with this project. :clap: 
### But kindly remove this file `rspec_results.html` its not needed for this project

**Pull request merged**